app/controllers/users_controller.rb

```diff
@@ -7,5 +7,13 @@ def withdrawal
   end
 
   def show
+    @user = User.new(id: params[:id])
+  end
+
+  def update
+    @user = User.find_by(id: params[:id])
```

Comment:
User.find_byだとnilが返りうるので、User.findで存在しない場合は例外を発生させましょう。

```suggestion
    @user = User.find(params[:id])
```